frontend/server/src/DAO/Identities.php

```diff
@@ -107,6 +107,81 @@ public static function findByUserId(int $userId): ?\OmegaUp\DAO\VO\Identities {
         return new \OmegaUp\DAO\VO\Identities($rs);
     }
 
+    /**
+     * @return array{identity: \OmegaUp\DAO\VO\Identities, isMainIdentity: bool, loggedAsMainIdentity: bool}|null
+     */
+    public static function resolveAssociatedIdentity(
+        string $usernameOrEmail,
+        \OmegaUp\DAO\VO\Identities $currentIdentity
+    ): ?array {
+        $sql = 'SELECT
+                    i.*,
+                    aut.acting_identity_id,
+                    aut.user_id AS acting_user_id,
+                    u.main_identity_id,
+                    e.email
+                FROM
+                    Identities i
+                INNER JOIN
+                    Users u
+                ON
+                    i.user_id = u.user_id
+                INNER JOIN
+                    Emails e
+                ON
+                    e.email_id = u.main_email_id
+                LEFT JOIN
+                    Auth_Tokens aut
+                ON
+                    aut.user_id = i.user_id
+                WHERE
+                    i.user_id = (
+                        SELECT
+                            si.user_id
+                        FROM
+                            Identities si
+                        INNER JOIN
+                            Users su
+                        ON
+                            si.user_id = su.user_id
+                        INNER JOIN
+                            Emails se
+                        ON
+                            se.email_id = su.main_email_id
```

Comment:
```suggestion
                            se.user_id = si.user_id
```

cualquier usuario debería poder usar cualquiera de sus correos. así también podemos eliminar `Users` de esta consulta!

Reply:
Esto se va, de acuerdo al comentario anterior